src/main/java/fi/okm/jod/yksilo/controller/profiili/TyopaikkaController.java

```diff
@@ -46,12 +47,12 @@ List<TyopaikkaDto> getAll(@AuthenticationPrincipal JodUser user) {
 
   @PostMapping
   @ResponseStatus(HttpStatus.CREATED)
```

Comment:
Technical specification delta: return status code 201 instead of 200

Reply:
won't fix, spec will be changed